etl/src/main/java/org/pih/hivmigration/etl/sql/MedpickupsMigrator.groovy

```diff
@@ -154,5 +186,6 @@ class MedpickupsMigrator extends SqlMigrator{
     def void revert() {
         executeMysql("drop table if exists hivmigration_dispensing_meds")
         executeMysql("drop table if exists hivmigration_dispensing")
+        clearTable("obs")
```

Comment:
Didn't we decide we wanted to try avoid clearing out all the obs if possible?  Or is that just too difficult to figure out how to do at this point? I could see the argument that this  becomes a bit of a rabbit hole.

I'm wondering if it would just be easier to create a two staging servers... one for migration, that gets wiped on each change to the migration pipeline, and another that *doesn't* have any migrated data it in but states stable and could be used for creating test data, etc... thoughts @dmdesimone  @cioan  @brandones  @mseaton ?

Reply:
Not sure why you'd want to create non-ephemeral test data. The only things we need to be testing is whether migrated-in data looks right and whether new data can be entered on top of it. I'd like to keep "migrating data into a database that has other data in it" out of our scope of work.

It hasn't caused any problems for my development process to just clear the obs table, since so far we don't have different Migrators creating interdependent obs. And I don't think we should.

Feel free if you want to be more exact about what gets reverted in the obs table, but also feel free not to?

Reply:
Yeah, I'm fine with just clearing out the obs table but then creating another Haiti EMR Test instance if we need to... @brandones  for the non-ephemeral question, I guess it's not just creating long-standing test data, but if @dmdesimone  is testing a new form on Haiti HIV Test and at the same time someone commits a change to the pipeline and his test data gets blown away from underneath him while he's in the process of testing it causes confusion.

Another option would be to *only* run the migration nightly, not on each commit, but then it's harder (or at least there's a longer turnaround) to testing migration steps.